bio/swarm/wrapper.py

```
@@ -3,10 +3,21 @@
 __license__ = "MIT"
 
 from snakemake.shell import shell
+from snakemake_wrapper_utils.snakemake import get_mem
 
 extra = snakemake.params.get("extra", "")
 log = snakemake.log_fmt_shell(stdout=True, stderr=True)
 
+# Define memory
+mem_mb = int(
+    get_mem(snakemake, "MiB", snakemake.params.get("mem_overhead_factor", 0.1))
```

Ruff (0.15.21)

[error] Undefined name `snakemake`

(F821)